src/Android/Autofill/AutofillHelpers.cs

```diff
@@ -41,6 +41,7 @@ public static class AutofillHelpers
             "com.amazon.cloud9",
             "com.android.browser",
             "com.android.chrome",
+            "com.android.htmlviewer",
```

Comment:
Does this need to be added to `SupportedBrowsers` above as well?

Reply:
idk, as I've said, it does not have a url bar, so I doubt if the accessibility service will work on this.

Reply:
**[2020/12]** Precision now added in `SupportedBrowsers` about the non-presence of this entry. See PR https://github.com/bitwarden/mobile/pull/1187.